src/test/unit/requests/subscriptionRequest.test.ts

```diff
@@ -2,103 +2,88 @@ import { SubscriptionRequests } from '../../../main/resources/requests/subscript
 const subscriptionManagementStub = sinon.stub(subscriptionManagementApi, 'post');
 
 describe(`getUserSubscriptions(${userIdWithSubscriptions}) with valid user id`, () => {
-  const userSubscriptions = subscriptionActions.getUserSubscriptions(userIdWithSubscriptions);
-  it('should return user subscription object', () => {
-    expect(userSubscriptions.courtSubscriptions.length).toBeGreaterThan(0);
-    expect(userSubscriptions.caseSubscriptions.length).toBeGreaterThan(0);
-  });
+  stub.withArgs(`/subscription/user/${userIdWithSubscriptions}`).resolves(subscriptionsData2);
 
-  it('should have mocked object in the case subscriptions list', () => {
-    expect(userSubscriptions.caseSubscriptions.filter((subscription) =>
-      subscription.reference === mockedCaseSubscription.reference).length).toBe(1);
-  });
+  it('should return user subscription object', async () => {
+    const userSubscriptions = await subscriptionActions.getUserSubscriptions(userIdWithSubscriptions) as Subscription;
 
-  it('should have mocked object in the court subscriptions list', () => {
-    expect(userSubscriptions.courtSubscriptions.filter((subscription) =>
-      subscription.name === mockedCourtSubscription.name).length).toBe(1);
-  });
-});
+    const subscription = userSubscriptions;
+    expect(subscription.caseSubscriptions.length).toEqual(2);
+    expect(subscription.courtSubscriptions.length).toEqual(3);
 
-describe(`getUserSubscriptions(${userIdWithoutSubscriptions}) with valid user id`, () => {
-  const userSubscriptions = subscriptionActions.getUserSubscriptions(userIdWithoutSubscriptions);
-  it('should return user subscription object', () => {
-    expect(userSubscriptions.courtSubscriptions.length).toBe(0);
-    expect(userSubscriptions.caseSubscriptions.length).toBe(0);
   });
-});
 
-describe(`non existing user Id getUserSubscriptions(${nonExistingUserId})`, () => {
-  const userSubscriptions = subscriptionActions.getUserSubscriptions(nonExistingUserId);
-  it('should return null', () => {
-    expect(userSubscriptions).toBe(null);
+  it('should have mocked object in the case subscriptions list', async () => {
+    const userSubscriptions = await subscriptionActions.getUserSubscriptions(userIdWithSubscriptions);
+    const subscription = userSubscriptions;
+
+    expect(subscription.caseSubscriptions[0].caseNumber).toBe(mockedCaseSubscription.reference);
   });
-});
 
-describe(`getSubscriptionByUrn(${validUrn}) with valid urn`, () => {
-  stub.withArgs('/hearings/urn/123456789').resolves({data: subscriptionsData});
-  it('should return hearing matching the urn', async () => {
-    const sub = await subscriptionActions.getSubscriptionByUrn(validUrn);
-    expect(sub.urn).toEqual(validUrn);
+  it('should have mocked object in the court subscriptions list', async () => {
+    const userSubscriptions = await subscriptionActions.getUserSubscriptions(userIdWithSubscriptions);
+    const subscription = userSubscriptions;
```

Comment:
what is the point of this const, you're just re-assigning it adds no value?

Reply:
removed